app/src/main/java/cz/quanti/razym/rocketapp/system/RocketListAdapter.kt

```diff
@@ -7,9 +7,14 @@ import cz.quanti.razym.rocketapp.databinding.RocketListItemBinding
 import cz.quanti.razym.rocketapp.model.Rocket
 
 class RocketListAdapter(
-    private val rockets: List<Rocket>
+    private val rockets: List<Rocket>,
+    private val itemListener: ItemListener
```

Comment:
je to detail ale tady bych preferoval to predat spis jako lambdu
```suggestion
    private val onItemClick: (Rocket) -> Unit
```

Reply:
to vypada lip, je to mnohem mene kodu, dik